INFO:__main__:PEX 03 start.
INFO:__main__:Connecting to device...
INFO:autopilot:ArduCopter V4.1.0-dev (26a3a02c)
INFO:autopilot:C22MKUTT
INFO:autopilot:Frame: QUAD
INFO:__main__:Device connected.
INFO:__main__:Looking for mission to execute...
INFO:__main__:Arming device...
INFO:__main__:Switching to GUIDED mode...
INFO:__main__:Waiting for arm...
INFO:autopilot:Arming motors
INFO:__main__:Device armed: True.
INFO:__main__:Device takeoff...
INFO:__main__:Current altitude: 0.008
INFO:__main__:Current altitude: 0.008
INFO:__main__:Current altitude: 0.008
INFO:__main__:Current altitude: 0.028
INFO:__main__:Current altitude: 0.331
INFO:__main__:Current altitude: 1.039
INFO:__main__:Current altitude: 1.909
INFO:__main__:Current altitude: 2.909
INFO:autopilot:EKF2 IMU0 in-flight yaw alignment complete
INFO:autopilot:EKF2 IMU1 in-flight yaw alignment complete
INFO:__main__:Current altitude: 3.973
INFO:__main__:Current altitude: 5.077
INFO:__main__:Current altitude: 6.213
INFO:__main__:Current altitude: 7.381
INFO:__main__:Current altitude: 8.558
INFO:__main__:Current altitude: 9.746
INFO:__main__:Current altitude: 10.947
INFO:__main__:Current altitude: 12.162
INFO:__main__:Current altitude: 13.38
INFO:__main__:Current altitude: 14.6
INFO:__main__:Current altitude: 15.825
INFO:__main__:Current altitude: 17.013
INFO:__main__:Current altitude: 18.055
INFO:__main__:Current altitude: 18.847
INFO:__main__:Current altitude: 19.352
INFO:__main__:Changing device mode from VehicleMode:GUIDED to AUTO...
INFO:autopilot:Mission: 1 Takeoff
INFO:autopilot:Mission: 2 WP
INFO:__main__:Device mode = VehicleMode:AUTO.
INFO:__main__:backing up old images...
INFO:root:Searching for target...
INFO:root:Starting camera feed...
INFO:root:configuring rgb stream.
INFO:root:Starting camera streams...
INFO:__main__:Program exception: ['Traceback (most recent call last):\n', '  File "C:/Users/C22Mason.Utt/Documents/GitHub/PEX3-COMP495/mission_starter_code.py", line 440, in <module>\n    conduct_mission()\n', '  File "C:/Users/C22Mason.Utt/Documents/GitHub/PEX3-COMP495/mission_starter_code.py", line 302, in conduct_mission\n    = check_for_initial_target(frame_display, net, classes)\n', '  File "C:/Users/C22Mason.Utt/Documents/GitHub/PEX3-COMP495/mission_starter_code.py", line 258, in check_for_initial_target\n    return (center_x, center_y), confidence, (x, y), radius, frame_display, b_boxes\n', "UnboundLocalError: local variable 'x' referenced before assignment\n"]
